cli/test_pack_python3.bats

```diff
@@ -1,6 +1,22 @@
 load '../test_helpers/bats-support/load'
 load '../test_helpers/bats-assert/load'
 
+skip_tests_if_st2_le_v3() {
+    # Utility function which skips tests if st2 is v3.0.0 or below. The py3 tests
+    # here require the fix https://github.com/StackStorm/st2/pull/4674 that will
+    # not be released until v3.0.1.
+
+    ST2_VER=$(st2 --version 2>&1)
+    ST2_VER=$(echo ${ST2_VER} | cut -d',' -f1)
+    ST2_VER=$(echo ${ST2_VER} | cut -d' ' -f2)
+    ST2_VER=$(echo ${ST2_VER} | sed -e "s/dev/.0/g")
+    ST2_VER=$(echo ${ST2_VER//.})
+
+    if [[ "${ST2_VER}" -le "300" ]]; then
+        skip "StackStorm is v3.0.0 or below"
```

Comment:
This will be shown in test output, and it's good practice to say why you are skipping it.

"Python 3 imports are broken on StackStorm < 3.0.1, skipping tests" is a better skip message.

You will also need to run this function from within the tests - see how `skip_tests_if_python3_is_not_available_or_if_already_running_under_python3` is called for guidance.

Reply:
Done